lib/CrEOF/Geo/String/Lexer.php

```diff
@@ -108,7 +108,7 @@ protected function getType(&$value)
     protected function getCatchablePatterns()
     {
         return array(
-            '[nesw\'",\X]',
+            '[nesw\'",X]',
```

Comment:
I don't think the `\X` should be unescape into `X`. Instead I think the original intention was to catch any character, including extented Unicode characters, as described in [PHP docs](https://www.php.net/manual/en/regexp.reference.unicode.php):

> In effect, this can be thought of as the Unicode equivalent of `.` [...]

However the doc also discourage using that because of impact on performance, and I don't see why we should be unicode aware in our use-case. And finally removing the sequence entirely still make all tests to pass (on PHP 7.2 and 7.3). So I suggest the following instead:

```suggestion
            '[nesw\'",]',
```